code/modules/cm_aliens/structures/fruit.dm

```diff
@@ -463,7 +511,11 @@
 		return
 	// Indicates the fruit is being picked, so other xenos can't eat it at the same time
 	F.picked = TRUE
-	if(!do_after(src, F.consume_delay, INTERRUPT_ALL, BUSY_ICON_FRIENDLY))
+	var/pick_delay = F.consume_delay
+	if(istype(src.strain, /datum/xeno_strain/gardener))
+		pick_delay /= 2
+	// Decrease time by half, if strain is gardener
+	if(!do_after(src, pick_delay, INTERRUPT_ALL, BUSY_ICON_FRIENDLY))
```

Comment:
should probably add this neat little bonus to strain description. also consider making gardeners feed it quicker